docs/conformance.md

```diff
@@ -18,6 +18,10 @@ VERSION=v0.33.0
 sonobuoy run --plugin https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/conformance.yaml
 ```
 
+To run without outside connectivity tests add the argument:
+
+`--e2e-skip "\[outside_connectivity\]"`
```

Comment:
sometimes you use `\\[` and sometimes `\[`

see please
https://github.com/kubevirt/kubevirt/pull/5169/files#diff-9a00de597476c98c8c1fc20ccdf64e20f43f7354733b4f56eca3d1aa229b4986R18

Reply:
when runing from the command line you should just use
`--e2e-skip "\[outside_connectivity\]"`
the `[` is escaped as it is a regexp but we are looking for the literall `[` sign.

in here: 
https://github.com/kubevirt/kubevirt/pull/5169/files#diff-9a00de597476c98c8c1fc20ccdf64e20f43f7354733b4f56eca3d1aa229b4986R18

`sonubuoy_args="$sonubouy_args --e2e-skip \"\\[outside_connectivity\\]\""`
the command is inside double quotes (`"`) therefor the next `"` needs to be escaped, next the escape char itself needs to be escaped so that the `[` will be escaped in turn. same considerations  after "_connectivity".

to show you what I mean I will  echo it from my terminal: 

```
[asadan@asadan kubevirt](fix_port-forward_no_explicit_ports)$ sonubuoy_args="$sonubouy_args --e2e-skip \"\\[outside_connectivity\\]\""
[asadan@asadan kubevirt](fix_port-forward_no_explicit_ports)$ echo $sonubuoy_args
--e2e-skip "\[outside_connectivity\]"
```

I hope it is clearer now.

Reply:
yep thanks, i mean that `--e2e-skip "\[outside_connectivity\]"`
would also work with double backslace maybe (need to check end 2 end, because its pretty confusing) ?
so it will be same style

Reply:
Can you write the command you think I should use?

Reply:
just to try `--e2e-skip "\\[outside_connectivity\\]"`
Its not crucial, just wondered if its better to have both of the places be the same,
but one is manual argument and the other is code, so its fine that it is as you did, as long as all is tested

Reply:
I am not sure it will work. I think I will need to get rid of the external double quotes for that.

Reply:
no need to bother then, thanks